### DISCUSSION TITLE
vscode插件很好看，如果粘贴图片能方便点就更好了

### DISCUSSION THREAD

**Discussion by liangyongrui** [1943time/inkdown#110] [Ideas]
vscode插件很好看，如果粘贴图片能方便点就更好了

比如 在同级目录下 自动保存 assert/xxxx.png

或者 工作区的固定目录下保存

不要让人再手动选择保存目录了

**Comment by 1943time** [comment]
嗯 下一版 回加上的

**Comment by 1943time** [comment]
v0.4.6可配置解决
